### PR TITLE
SEO optimisations including social card, landing page title and description, and relevant config changes

### PR DESCRIPTION
**Pull request by axmmisaka**
This PR
1. Created a homemade 1200x630 SVG social card for display
2. Changed the config to use that SVG
3. Changed standalone pages' (index, community and publication) `title` and `description` field to allow social medias to generate more informative description

Preview:
图片
图片

### CODE REVIEW

**Comment by axmmisaka** [comment]
> This is great! 🚀 What about the warning message about image size?

idk; it's an SVG so it might cause some issue (I don't think the actual size of the canvas will be read); we could replace with a better-designed png?

**Comment by lhstrh** [comment]
studio-343b4556219b98e2c2916dcfbe94e9aa-cgh9m7cq

File: src/pages/index.tsx [hunk 1]
```diff
@@ -146,8 +146,8 @@ export default function Home(): JSX.Element {
   const { siteConfig } = useDocusaurusContext();
   return (
     <Layout
-      title={`Welcome`}
-      description="Intuitive concurrent programming in any language"
+      title="Intuitive concurrent programming in any language"
+      description="Build time-sensitive, concurrent, and distributed systems effortlessly"
```
Comment (lhstrh) [inline src/pages/index.tsx:150]:
```suggestion
      description="Build time-sensitive, concurrent, and distributed systems — effortlessly"
```

**Comment by lhstrh** [comment]
Not sure how to test it, but I guess it won't hurt if we just merge this...

**Review by lhstrh — APPROVED** [review]

**Pull request merged**